core/src/com/mygdx/crazybomber/Map.java

```diff
@@ -0,0 +1,195 @@
+         player = new Player(0f, 0f, playerTexture);
+        player.setSize(50,50);
+        player.setPosition(615,600);
+
+
+        stage = new Stage();
+        Gdx.input.setInputProcessor(stage);
+        atlas = new TextureAtlas("ui/button.pack"); //defines the region of sprite image
+        skin = new Skin(atlas); //using this for the text button
+        table = new Table(skin); //table with the skin's atlas
+        //table.setBounds(500, 300, Gdx.graphics.getWidth(), Gdx.graphics.getHeight()); //sets the bounds to full screen
+        table.setBounds(1050, 600, 100,100 ); // coordinates of the table, size
+
+
+        white = new BitmapFont(Gdx.files.internal("font/white.fnt"), false); //this false is so the chars aren't flipped
+        black = new BitmapFont(Gdx.files.internal("font/black.fnt"), false);
+
+        TextButton.TextButtonStyle textButtonStyle = new TextButton.TextButtonStyle();
+        textButtonStyle.up = skin.getDrawable("button.up");
+        textButtonStyle.down = skin.getDrawable("button.down");
+        textButtonStyle.pressedOffsetX = 1; //when button is pressed, text moves to right by 1
+        textButtonStyle.pressedOffsetY = -1;
+        textButtonStyle.font = black;
+
+        buttonBack = new TextButton("BACK", textButtonStyle);
+        buttonBack.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                ((Game) Gdx.app.getApplicationListener()).setScreen(stateRepository.peek());
+            }
+        });
+
+
+        LabelStyle headingStyle = new LabelStyle(white, Color.WHITE);
+        heading =  new Label(BombermanWithFriends.TITLE, headingStyle);
+        heading.setFontScale(2); //doubles the font size
+        table.add(heading);
+        table.getCell(heading).spaceBottom(10);
+        table.row();
+        table.padBottom(1);
+        table.row();
+        table.add(buttonBack);
+        table.debug();
+
+        stage.addActor(table);
+    }
+
+
+
+    @Override
+    public void render(float delta) {
+        Gdx.gl.glClearColor(0,0,0,1);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+        stage.act(delta);
+        batch.begin();
+        splash.draw(batch);
+
+
+        isPressedUP = Gdx.input.isKeyPressed(Input.Keys.UP);
+        isPressedDOWN = Gdx.input.isKeyPressed(Input.Keys.DOWN);
+        isPressedRIGHT = Gdx.input.isKeyPressed(Input.Keys.RIGHT);
+        isPressedLEFT =  Gdx.input.isKeyPressed(Input.Keys.LEFT);
+        isPressedW = Gdx.input.isKeyPressed(Input.Keys.W);
+        isPressedS = Gdx.input.isKeyPressed(Input.Keys.S);
+        isPressedD = Gdx.input.isKeyPressed(Input.Keys.D);
+        isPressedA = Gdx.input.isKeyPressed(Input.Keys.A);
+        isPressedSHIFT = Gdx.input.isKeyPressed(Input.Keys.SHIFT_LEFT); //run
+        isPressedSPACE = Gdx.input.isKeyPressed(Input.Keys.SPACE); //runevenfaster
+        char direction = '\0';
```

Comment:
make a TODO comment, on how we need to change this move function

Reply:
made TODO: change move function